onnxruntime/core/optimizer/double_qdq_pairs_remover.cc

```diff
@@ -81,41 +98,51 @@ bool DoubleQDQPairsRemover::IsNodeRemovable(
       grandchild->OpType() != "DequantizeLinear") {
     return false;
   }
+
   const auto get_constant_initializer = [&graph](const std::string& initializer_name) {
     return graph.GetConstantInitializer(initializer_name, true);
   };
+
+  // Each QDQ pair (i.e., parent -> self, child -> grandchild) has to meet the following additional requirements:
+  // - Scalar/constant zero-point and scale.
+  // - The DQ and Q ops within a pair must have the same scale and zero-point.
+  //   However, each pair is allowed to have different scales and zero-points.
+  //
+  // TODO(adrianlizarraga): IsQDQPairSupported() requires an explicit zero-point input, but technically a default
```

Comment:
>adrianlizarraga [](start = 10, length = 15)

git history/blame is enough to track this. no need to have personal contact here.

Reply:
The automated linter warns about leaving user id information with every TODO. Should these be ignored?

Reply:
![image](https://github.com/microsoft/onnxruntime/assets/19691973/b73835ef-bbc7-4b98-b99a-d00db500e5c2)
 This is what I mean.

Reply:
OK, seems new rules to keep track of these TODOs. Please ignore my comments. :D

---
In reply to: [1317610216](https://github.com/microsoft/onnxruntime/pull/17015#discussion_r1317610216) [](ancestors = 1317610216)

Reply:
> git history/blame is enough to track this. no need to have personal contact here.

FWIW, I agree and have been ignoring these types of lint issues.